tests/foreman/ui/test_activationkey.py

```diff
@@ -795,7 +793,68 @@ def test_positive_update_activation_key_4(self, cv2_name):
                                  (common_locators["alert.success"]))
             selected_cv = self.activationkey.get_attribute(name, cv_locator)
             self.assertEqual(cv2_name, selected_cv)
-            # TODO: Need to check for RH Product too
+
+    @run_only_on('sat')
+    @data(*valid_data_list())
+    def test_positive_update_activation_key_with_rh_product(self, cv2_name):
+        """@Test: Update Content View in an Activation key
+
+        @Feature: Activation key - Positive Update
+
+        @Steps:
+        1. Create Activation key
+        2. Update the Content view with another Content view
+        which has RH products
+
+        @Assert: Activation key is updated
+
+        """
+        name = gen_string("alpha", 8)
+        env1_name = gen_string("alpha", 8)
+        env2_name = gen_string("alpha", 8)
+        cv1_name = gen_string("alpha", 8)
+        rh_repo1 = {
+            'name': ('Red Hat Enterprise Virtualization Agents for RHEL 6 '
+                     'Server RPMs x86_64 6Server'),
+            'product': 'Red Hat Enterprise Linux Server',
+            'reposet': ('Red Hat Enterprise Virtualization Agents '
+                        'for RHEL 6 Server (RPMs)'),
+            'basearch': 'x86_64',
+            'releasever': '6Server',
+        }
+        rh_repo2 = {
+            'name': ('Red Hat Enterprise Virtualization Agents for RHEL 6 '
+                     'Server RPMs i386 6Server'),
+            'product': 'Red Hat Enterprise Linux Server',
+            'reposet': ('Red Hat Enterprise Virtualization Agents '
+                        'for RHEL 6 Server (RPMs)'),
+            'basearch': 'i386',
+            'releasever': '6Server',
+        }
+        org = entities.Organization().create()
+        manifest_path = manifests.clone()
+        entities.Organization(id=org.id).upload_manifest(path=manifest_path)
```

Comment:
Is it really creating new organization here despite ID was specified?
But yea, your suggestion is better in any case.